### PR TITLE
A few JS fixes in the admin

### PR DESCRIPTION
**Pull request by jhawthorn**
* Use text field instead of hidden field for variant autocomplet (fixes firefox issue)
* Fix initSelection of variant_autocomplete
* Fixes case in stock_movement.js where URL was generated with `undefined`
* Remove AdaptiveMenu.js

### CODE REVIEW

File: backend/app/assets/javascripts/spree/backend/admin.js.erb [hunk 1]
```diff
@@ -18,15 +18,6 @@ jQuery(function($) {
   // Vertical align of checkbox fields
   $('.field.checkbox label').vAlign()
 
-  <% # Re-adjusting admin menu during test causes tests to fail,
-     # like states_spec and shipping_methods_spec. Let's not do this. %>
-  <% unless Rails.env.test? %>
```
Comment (gmacdougall) [inline backend/app/assets/javascripts/spree/backend/admin.js.erb]: Ugh... :fireworks: :+1: for getting rid of terrible stuff like this.

**Comment by gmacdougall** [comment]
:+1:

**Comment by magnusvk** [comment]
:+1:

**Comment by cbrunsdon** [comment]
Giant :+1:. :burnitdown:

**Pull request merged**